src/components/Layout/Atoms.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import cn from 'classnames'
+
+import styles from './Layout.module.css'
+
+const PageContainer = ({ className, children }) => (
+  <div className={cn(styles.page_container, className)}>
+    {children}
+  </div>
+)
+
+const MainContainer = ({ className, children }) => (
+  <main className={cn(styles.main_container, className)}>
+    {children}
+  </main>
+)
+
+const ContentContainer = ({ className, children }) => (
+  <div className={cn(styles.content_container, className)}>
+    {children}
+  </div>
+)
+
+// all containers will have the same two proptypes
+const containerPropTypes = {
+  className: PropTypes.string,
+  children: PropTypes.node.isRequired
+}
+
+PageContainer.propTypes = { ...containerPropTypes }
+MainContainer.propTypes = { ...containerPropTypes }
+ContentContainer.propTypes = { ...containerPropTypes }
+
+export {
+  ContentContainer,
+  MainContainer,
+  PageContainer
+}
```

Comment:
The 'Atom' concept comes from [Atomic Design](https://atomicdesign.bradfrost.com/chapter-2/), which I don't want to implement beyond the concept of 'atoms' (the whole atoms -> molecules -> organisms thing is overkill for us).

'Atom' components are just wrappers that apply some styling. `ContentContainer`, `MainContainer`, and `PageContainer` merge any passed-in CSS classes with their own CSS classes and render children.